Meadowlark.net/Meadowlark.Net.Core/Middleware/ValidateDocumentMiddleware.cs

```diff
@@ -0,0 +1,30 @@
+using Meadowlark.Net.Core.Model;
+using static Meadowlark.Net.Core.Validation.DocumentValidator;
+
+namespace Meadowlark.Net.Core.Middleware;
+
+public static class ValidateDocumentMiddleware
+{
+  /**
+   * Validates JSON document shape
+   */
+  public static async Task<MiddlewareModel> DocumentValidation(MiddlewareModel middlewareModel)
```

Comment:
I got a warning when building the solution because of this function being async but not having any awaits:

`warning CS1998: This async method lacks 'await' operators and will run synchronously.`

Reply:
Some of the 3rd-party libs I tried wanted await. Current one doesn't but I'm going to "wait" on changing this. There will other awaits in the middleware with the DB stuff but removing this one first cascades errors in several places.